students/HABTAMU/lesson08/assignment/inventory.py

```diff
@@ -0,0 +1,62 @@
+"""
+Assignment 8 Norton Furniture Assignment
+Inventory Management System
+"""
+import csv
+from functools import partial
+
+def add_furniture(invoice_file,
+                  customer_name,
+                  item_code,
+                  item_description,
+                  item_monthly_price):
+    """
+    This will create invoice_file if it doesn’t exist or append a new line to it if it does. 
+    :param invoice_file
+    :param customer_name
+    :param item_code
+    :parm item_description
+    :param item_monthly_price
+    """
+
+    with open(invoice_file, 'a+') as csvfile:
+        csvfile.write(f'{customer_name}, {item_code}, {item_description}, {float(item_monthly_price):.2f}\n')
+
+
+def single_customer(customer_name, invoice_file):
+    """
+    This will load individual customers rental.
+    :param invoice_file
+    :param customer_name
+    """
+    customer_furniture = partial(add_furniture, invoice_file=invoice_file, customer_name=customer_name)
+
+    FIELD_NAME = ['customer_name', 'item_code', 'item_description', 'item_monthly_price']
```

Comment:
I think this FIELD_NAME list is not necessary. You are not using it anywhere. Otherwise looks good. Nice closure, and you are using partial to change the signature of add_furniture. In your doc string, I would include information about the return, and how to use it (returns a function with the following inputs, purpose).